students/itsabhi/Lesson3/mailroom.py

```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/env python3
+
+
+def sending_thanks(d):
+    donor_hist = d
```

Comment:
There's no need to reassign the `d` argument here, but I would use `donor_hist` as your parameter name as it's more meaningful/readable